solidity/dashboard/src/css/coverage-pools-page.less

```diff
@@ -1,59 +1,63 @@
-.coverage-pool__checklist {
-  max-width: unset;
-  background-color: @color-mint-20;
-  padding: 2rem;
-  margin-bottom: 1rem;
-  border-radius: 10px;
-
-  .banner__title {
-    &:extend(.h3);
-  }
+.coverage-pool__metrics {
+  display: grid;
+  grid-template-areas:
+    "tvl  tvl tvl"
+    "reward-rate  total-rewards lifetime-covered";
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: auto auto;
+  gap: 1rem;
 
-  .banner__close-icon {
-    width: 16px;
-    height: 16px;
-    top: 16px;
-    right: 16px;
-  }
+  .metrics {
+    &__tvl {
+      grid-area: tvl;
 
-  .checklist-wrapper {
-    display: flex;
-    flex-wrap: wrap;
+      .tvl.tvl--fiat-currency {
+        color: @color-grey-40;
+        font-weight: 600;
+        margin-top: 0.75rem;
+        margin-left: 2.25rem;
```

Comment:
Can we add a little bit more left-margin here?

![image](https://user-images.githubusercontent.com/40306834/127637416-5dd6dd24-88ee-4cec-8976-ac92066cc2c7.png)

The dollar sine should start be placed exactly under the first digit of the amount.